### PR TITLE
Support synchronization of event table from time graph selection

### PR DESCRIPTION
**Pull request by PatrickTasse**
Signed-off-by: Patrick Tasse <patrick.tasse@ericsson.com>

### CODE REVIEW

File: packages/react-components/src/components/table-output-component.tsx [hunk 1]
```diff
@@ -2,10 +2,11 @@
 import { AbstractOutputComponent, AbstractOutputProps, AbstractOutputState } from './abstract-output-component';
 import * as React from 'react';
 import { AgGridReact } from 'ag-grid-react';
-import { ColDef, IDatasource, GridReadyEvent, CellClickedEvent } from 'ag-grid-community';
+import { ColDef, IDatasource, GridReadyEvent, CellClickedEvent, GridApi, ModelUpdatedEvent } from 'ag-grid-community';
```
Comment (PatrickTasse) [inline packages/react-components/src/components/table-output-component.tsx:5]: ModelUpdatedEvent is unused

File: packages/react-components/src/components/table-output-component.tsx [hunk 2]
```diff
@@ -49,6 +53,10 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 }
             }
         };
+
+        this.props.unitController.onSelectionRangeChange(range => { this.handleTimeSelectionChange(range); });
```
Comment (PatrickTasse) [inline packages/react-components/src/components/table-output-component.tsx:57]: This handler is never removed from the unit controller when the table component is unmounted, so it continues to be called for closed event tables.

File: packages/react-components/src/components/table-output-component.tsx [hunk 3]
```diff
@@ -186,4 +194,50 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
...
+        const model = lineResponse.model;
+        const lines = model.lines;
+        if (lines.length === 0) {
+            return undefined;
+        }
+        return lines[0].index;
+    }
+
+    private onModelUpdated = async () => {
+         if (this.pendingIndex && this.gridApi) {
```
Comment (PatrickTasse) [inline packages/react-components/src/components/table-output-component.tsx:236]: Wrong indentation

**Comment by MatthewKhouzam** [comment]
Doesn't seem to work, are there other dependencies?

**Review by MatthewKhouzam — APPROVED** [review]
Works on my machine, code looks good.

**Review by bhufmann — APPROVED** [review]
Looks good to me. I tested it and it's a great improvement for navigating in the table using a timegraph view.

**Pull request merged**